lib/gis/parser_rest.c

```diff
@@ -0,0 +1,292 @@
+/*!
+   \file lib/gis/parser_rest.c
+
+   \brief GIS Library - Argument parsing functions (reStructuredText output)
+
+   (C) 2012 by the GRASS Development Team
+
+   This program is free software under the GNU General Public License
+   (>=v2). Read the file COPYING that comes with GRASS for details.
+
+   \author Luca Delucchi
+ */
+#include <stdio.h>
+#include <string.h>
+
+#include <grass/gis.h>
+#include <grass/glocale.h>
+
+#include "parser_local_proto.h"
+
+static void print_escaped_for_rest(FILE *f, const char *str);
+static void print_escaped_for_rest_options(FILE *f, const char *str);
+
+/*!
+   \brief Print module usage description in reStructuredText format.
+ */
+void G__usage_rest(void)
```

Comment:
Why are the names inconsistent now?

Reply:
The API is the same in both cases G__usage_rest and G__usage_markdown. The only difference is that G__usage_markdown calls static usage_md while G__usage_rest provides the implementation directly. I just kept code structure which is also common elsewhere, but it is not needed here now. I can clean that up so that there is no jump from the API function to static function and there is only one function. That will raise less questions.